lib/iex/lib/iex.ex

```diff
@@ -255,6 +255,28 @@ defmodule IEx do
   Number of expressions and their results to keep in the history.
   The value is an integer. When it is negative, the history is unlimited.
 
+  ## Documentation Helpers
+
+  Documentation Helpers allows iex to have a configurable set of modules
```

Comment:
It should read "Documentation Helpers allow IEx"
(change the tense of the "to allow" verb and capitalization of IEx)